install/action.yaml

```diff
@@ -50,35 +57,39 @@ runs:
       with:
         fetch-depth: "100"
     - run: git fetch --prune --tags --unshallow
-      shell: bash -l {0}
+      shell: bash -el {0}
     - uses: actions/setup-python@v4
       with:
         python-version: ${{ inputs.python-version }}
     - uses: conda-incubator/setup-miniconda@v2
       with:
         miniconda-version: "latest"
-    - if: inputs.conda-update == 'true'
-      run: |
-        conda update -c defaults -n base conda
-      shell: bash -l {0}
+        auto-update-conda: ${{ inputs.conda-update }}
+        # use-only-tar-bz2: ${{ inputs.cache }}  # Does not give latest version
     - run: echo "TODAY=$(date +'%Y-%m-%d')" >> $GITHUB_ENV
-      shell: bash -l {0}
+      shell: bash -el {0}
     - if: runner.os == 'Windows'
       run: echo "ENVS_PATH=${{ format('{0}\{1}', env.CONDA, 'envs') }}" >> $GITHUB_ENV
-      shell: bash -l {0}
+      shell: bash -el {0}
     - if: runner.os != 'Windows'
       run: echo "ENVS_PATH=${{ format('{0}/{1}', env.CONDA, 'envs') }}" >> $GITHUB_ENV
-      shell: bash -l {0}
+      shell: bash -el {0}
     - if: inputs.cache == 'true'
       uses: actions/cache@v3
       with:
         path: ${{ env.ENVS_PATH }}
-        key: ${{ inputs.name  }}-${{ runner.os }}-conda-${{ inputs.python-version }}-${{ hashFiles('setup.py') }}-${{ env.TODAY }}
+        key: ${{ inputs.name  }}-${{ runner.os }}-${{ inputs.conda-mamba }}-${{ inputs.python-version }}-${{ hashFiles('setup.py') }}-${{ env.TODAY }}
       id: cache
+    - if: inputs.conda-mamba == 'mamba' && steps.cache.outputs.cache-hit != 'true'
+      run: |
+        echo "MAMBA_NO_BANNER=1" >> $GITHUB_ENV
+        conda install mamba -c conda-forge -n base
+        conda run -n base mamba init --all
```

Comment:
Ok I didn't know you have to run `mamba init`. Funny to see conda executing mamba!